src/Aspire.Cli/Configuration/ConfigurationService.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Aspire.Cli.Configuration;
 
-internal sealed class ConfigurationService(IConfiguration configuration, DirectoryInfo currentDirectory, FileInfo globalSettingsFile) : IConfigurationService
+internal sealed class ConfigurationService(IConfiguration configuration, CliExecutionContext executionContext, FileInfo globalSettingsFile) : IConfigurationService
```

Comment:
Changes in this file are just tidy-up around working directory handling, not related to the main purpose of this PR.